extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -476,11 +482,15 @@ export class SafeframeHostApi {
          (expandWidth > this.creativeSize_.width ||
           expandHeight > this.creativeSize_.height))) {
       dev().error(TAG, 'Invalid expand values.');
+      this.sendResizeResponse(
+          /* SUCCESS? */ false, SERVICE.EXPAND_RESPONSE);
```

Comment:
I don't see any tests for the new `sendResizeResponse` behavior

(here and 2x below)

Reply:
done